### DISCUSSION TITLE
Unknown column x.TABLE_NAME when update database

### DISCUSSION THREAD

**Discussion by hous04** [symfony/symfony#51861] [Q&A]
### Symfony version(s) affected

6.3.5

### Description

**SOLVED**
Sorry, the error comes from serverVersion "10.11.2-MariaDB" in the database_url in .env file, I should rename serverName

DATABASE_URL="mysql://***:**@127.0.0.1:3306/db_name?serverVersion=10.11.2-MariaDB&charset=utf8mb4"

I have upgraded the version of tow projects from 6.3.x to 6.3.5, in the tow projects I get this error when I try to update the database or make migration

`
In ExceptionConverter.php line 67:
                                                                                                                                       
  An exception occurred while executing a query: SQLSTATE[42S22]: Column not found: 1054 Unknown column 'x.TABLE_NAME' in 'on clause'  
                                                                                                                                       

In Exception.php line 28:
                                                                                        
  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'x.TABLE_NAME' in 'on clause'  
                                                                                        

In Statement.php line 130:
                                                                                        
  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'x.TABLE_NAME' in 'on clause' `

The error comes from  **serverVersion** "10.11.2-MariaDB" in the **database_url** in .env file, I should remove serverName 

`DATABASE_URL="mysql://***:**@127.0.0.1:3306/db_name?serverVersion=10.11.2-MariaDB&charset=utf8mb4"`

### How to reproduce

`php bin/console make:migration`

**Comment by kai-zer-ru** [comment]
Hi! Have you solved this problem?

**Reply by ghost** [reply]
He edited the description and said he had solved the problem.

**Reply by hous04** [reply]
I have changed the serverVersion in ..env file in DATABASE_URL
in my case replaced by **serverVersion=8.0.34-0ubuntu0.22.04.1**
